biosimulations/apps/dispatch-api/src/simulation-run/simulation-run.module.ts

```diff
@@ -28,7 +34,21 @@ import { SimulationRunService } from './simulation-run.service';
         schema: SimulationFileSchema,
       },
     ]),
+    
+  ],
+  providers: [
+    SimulationRunService,
+    {
+      provide: 'DISPATCH_MQ',
```

Comment:
This can be separated into it own module. 
@gmarupilla  don't worry about addressing this, I can work on it.